app/styles/main.css

```diff
@@ -783,6 +784,7 @@ section{word-break:break-word;}
   h1.hero-head{font-size:1.65rem;}
   .hero{padding-top:3%;padding-bottom:75px;}
   .hero-lead{line-height:1.125rem;}
+  .no-mobile-paddings{padding-left:0px; padding-right: 0px;}
```

Comment:
Let's remove the class `.no-mobile-paddings` as this can be resolved by adding the `no-padding` classname to `line 9`.